web/tests/integration/app.test.ts

```diff
@@ -0,0 +1,122 @@
+import { gql } from "@apollo/client";
+
+import {
+  integrationDBExecuteQuery,
+  integrationDBSetup,
+  integrationDBTearDown,
+} from "./setup";
+
+import { getAPIUserClient } from "./test-utils";
+
+// TODO: Consider moving this to a generalized jest environment
+beforeEach(integrationDBSetup);
+beforeEach(integrationDBTearDown);
+
+describe("user role", () => {
```

Comment:
same as with team, ideally we would do this through the next.js GraphQL endpoint, to make sure tampering with the team ID header doesn't do anything

Reply:
we test an invalid team being passed in now